Automattic/wp-calypso
# chore(@automattic/page-template-modal): Use the same dependencies than ETK

#### Changes proposed in this Pull Request

* [ETK] Replace `"*"` version ranges with the actual range extracted from `yarn.lock`
* [@automattic/page-template-modal] Use the same versions than ETK

#### Testing instructions

* Note that the `dist` files produced by ETK have not changed. To verify: build ETK in this branch and trunk, and diff all files inside `dist/**/*` folders. Example: `for file in apps/editing-toolkit/**/dist/**/*; do diff $file ../wp-calypso-trunk/$file; done`
